app/src/main/res/layout/fragment_login.xml

```diff
@@ -41,12 +41,12 @@
 
             <android.support.design.widget.TextInputEditText
                 android:id="@+id/username"
-                android:hint="@string/hint_username"
-                android:paddingLeft="0dp"
+                style="@style/EditText"
```

Comment:
make sure the style attribute is at the end of the declaration
.
.
android:hint="showcase"
style="@style/stylename"/>